src/_pytest/mark/structures.py

```diff
@@ -530,7 +530,7 @@ def __getattr__(self, name: str) -> MarkDecorator:
                 warnings.warn(
                     "Unknown pytest.mark.%s - is this a typo?  You can register "
                     "custom marks to avoid this warning - for details, see "
-                    "https://docs.pytest.org/en/stable/mark.html" % name,
+                    "https://docs.pytest.org/en/latest/how-to/mark.html" % name,
```

Comment:
Note how this already points to `stable/` (with a link that works correctly) rather than `latest/`, see 93d2ccbfb7b99755e8f4a2a3947c466949ee3c46. So I'm not sure where @bsypniewski got the `latest` link from...

I believe we should continue to point to `stable/` here, but to `stable/how-to/mark.html` which will then work correctly once 7.0.0 is out.

What do others think?

Reply:
Actually, that's a problem for far more than just this link... I opened #8831 to track this. As for this one, I believe the old URL is actually correct for now.

Reply:
> I believe we should continue to point to stable/ here, but to stable/how-to/mark.html which will then work correctly once 7.0.0 is out.

Makes sense. I found the new page searching in `latest/` but the link should point to `stable/` like before.

Reply:
Link updated in https://github.com/pytest-dev/pytest/pull/8830/commits/820d56c6a8957164ab5872879d3b411b209701f3.